api/storage.go

```diff
@@ -0,0 +1,10 @@
+package api
+
+type Storage struct {
+	Name          string
+	Driver        string
+	DriverOptions map[string]string
+	Source        string
+	Fstype        string
+	MoutOption    []string
+}
```

Comment:
This `Storage` is partly covered by virtcontainers `Mount` but some fields are missing, and the name might be misleading. A few questions:

- In which context is this structure used?
- How is it different than a generic container Mount
- What is the `Driver` field about? Do you have one example to share?